app/estimate_square_size_test.go

```diff
@@ -88,7 +88,7 @@ func Test_estimateSquareSize_MultiBlob(t *testing.T) {
 			"100 multiblob single share transaction", func() [][]int {
 				return [][]int{blobfactory.Repeat(int(100), 100)}
 			},
-			16, 5,
+			16, 11,
```

Comment:
[simple comment]
perhaps we can eventually include some some option to use a multiblob commit and no namespaces to be more efficient in the future